examples/rosa-classic-with-unmanaged-oidc/main.tf

```diff
@@ -35,11 +35,11 @@ module "operator_roles" {
   oidc_endpoint_url    = module.oidc_provider.oidc_endpoint_url
 }
 
-// module "vpc_private" {
-//   source = "../../modules/vpc-private"
-// 
-//   name = "${var.cluster_name}-vpc"
-// }
+module "vpc" {
```

Comment:
I think the example name should include BYO VPC

Reply:
Lets push this in another PR